### ISSUE TITLE
Allow for changing webservice url during runtime?

### ISSUE THREAD

**Issue by anneferger** [Edirom/vife-meigarage-webclient#31]
* now added as a buildtime variable
* how to change so that https://dev.meigarage.edirom.de/ uses the webservice at https://dev.meigarage.edirom.de/ege-webservice?

**Comment by peterstadler** [comment]
Sounds like a plan :) 
I found https://stackoverflow.com/questions/53010064/pass-environment-variable-into-a-vue-app-at-runtime which lists at least two options; the more hacky one (replacing strings in the js files with an entrypoint script)  I've also seen (done) at https://github.com/TEIC/romajs/blob/dev/docker-entrypoint.sh

**Comment by anneferger** [comment]
I thought some more about this, and I think what would be the easiest is to set the webservice_url variable to dev.meigarage.edirom.de when the action is running on the dev branch and not the main, see https://github.com/Edirom/vife-meigarage-webclient/tree/feature/set-webservice-url
- we don't need to change the variable during runtime this way, only during build

**Comment by peterstadler** [comment]
Yes, that will work for us – but still the build artifact ist tied to the deployment, hence it won't work for others (e.g. to be deployed at meigarage.example.com).

**Comment by anneferger** [comment]
Yes, good point! We should still have this option
